src/style.css

```diff
@@ -0,0 +1,100 @@
+@import url("https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap");
+
+html {
+  font-family: Roboto, sans-serif;
+  font-size: 10px;
+  color: #060b35;
```

Comment:
Maybe use css variable for common colors?

Reply:
yes, great idea. i have fixed that